pytorch/inference/docker/build_artifacts/entrypoint.sh

```diff
@@ -62,11 +62,12 @@ printf "Model path: %s\n" "$MODEL_PATH"
 printf "TS_CONFIG: %s\n" "$TS_CONFIG"
 # Start the Model Server
 if [[ -z "$MODEL_PATH" ]]; then
-  torchserve --start --ts-config /home/model-server/config.properties --model-store /opt/ml/model &
+  torchserve --start --ts-config /home/model-server/config.properties --model-store /opt/ml/model
```

Comment:
I did a quick search through this directory, and it looks like this file is only used for neuron images, but it's unclear from the directory location and naming convention. Would it be possible to rename it to `neuron-entrypoint.sh` - or something that signals that this is for neuron images?

Reply:
Done.